src/backend/database/migrations/micropowermanager/2020_12_23_165814_create_map_settings_table.php

```diff
@@ -19,7 +19,6 @@ public function up() {
             $table->double('latitude', 10);
             $table->double('longitude', 10);
             $table->string('provider')->nullable();
-            $table->string('bingMapApiKey')->nullable();
             $table->timestamps();
```

Comment:
Generally, I'm not a fan of changing migration files retrospectively. 

This will create confusion in the long run. For example, this change (i.e. the removal of column `bingMapApiKey`) will not be applied to existing instances of MPM. Only new instances will have the column removed.

I think it's always better to create a new migration file for this, this also allows us to roll back changes if need be.

Could you send a new PR that restores this file and create a new migration to remove the column? You can use `php artisan make:migration-tenant` to get a template of the migration.
`

Reply:
I went with the assumption there will be breaking changes that require us to do a fresh migration. Not a good approach, I must admit. 

 
> Could you send a new PR that restores this file and create a new migration to remove the column? You can use php artisan make:migration-tenant to get a template of the migration.

Will apply a new migration accordingly, thanks.